<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1151</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1151</width>
      <height>701</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>200</y>
        <width>381</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Gestion client</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>350</y>
        <width>381</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Gestion commande</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>10</y>
        <width>1141</width>
        <height>681</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-image: url(:/images/11878958_Online_Shoping_29.png);
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>11878958_Online_Shoping_29.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>label</zorder>
     <zorder>pushButton_5</zorder>
     <zorder>pushButton_6</zorder>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>440</y>
        <width>1071</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>100</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ID_CLIENT</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>170</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>NOM</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>220</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>PRENOM</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>270</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>MAIL</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ID_CLIENT">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>100</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="NOM">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>160</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="PRENOM">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>210</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="MAIL">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>270</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>340</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>SEXE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>390</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>100</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>100</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>30</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>retour</string>
      </property>
     </widget>
     <widget class="QTableView" name="statsCl">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>170</y>
        <width>341</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>30</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Gestion des clients</string>
      </property>
     </widget>
     <widget class="QPushButton" name="statbtnC_2">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>140</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>stats</string>
      </property>
     </widget>
     <widget class="QComboBox" name="SEXE">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>340</y>
        <width>111</width>
        <height>26</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>homme</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>femme</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>autre</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>400</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="triCLB">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>400</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>trier</string>
      </property>
     </widget>
     <widget class="QComboBox" name="triCL">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>400</y>
        <width>121</width>
        <height>26</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>ASC</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DESC</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>9</y>
        <width>1121</width>
        <height>691</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>11878958_Online_Shoping_29.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>label_14</zorder>
     <zorder>tableView</zorder>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>label_4</zorder>
     <zorder>label_5</zorder>
     <zorder>ID_CLIENT</zorder>
     <zorder>NOM</zorder>
     <zorder>PRENOM</zorder>
     <zorder>MAIL</zorder>
     <zorder>label_6</zorder>
     <zorder>pushButton</zorder>
     <zorder>pushButton_2</zorder>
     <zorder>pushButton_3</zorder>
     <zorder>pushButton_7</zorder>
     <zorder>statsCl</zorder>
     <zorder>label_12</zorder>
     <zorder>statbtnC_2</zorder>
     <zorder>SEXE</zorder>
     <zorder>lineEdit</zorder>
     <zorder>triCLB</zorder>
     <zorder>triCL</zorder>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>350</y>
        <width>1111</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ID_COMMANDE</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ID_COMMANDE">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>40</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="MONTANT">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>90</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>MONTANT</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>150</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>STATUT</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>200</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>DATE_COMMANDE</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>250</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>nom_client</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>310</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>40</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_14">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>40</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="DATE_COMMANDE">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>200</y>
        <width>111</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>20</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>retour</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>250</y>
        <width>111</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="STATUT">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>150</y>
        <width>111</width>
        <height>26</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>livree</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>en cours</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>annulee</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>0</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Gestion des commandes</string>
      </property>
     </widget>
     <widget class="QTableView" name="stats_commande">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>90</y>
        <width>341</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="statbtnC">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>60</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>stats</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>310</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="triCB">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>310</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>trier</string>
      </property>
     </widget>
     <widget class="QComboBox" name="triC">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>310</y>
        <width>111</width>
        <height>26</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>ASC</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DESC</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>-1</y>
        <width>1141</width>
        <height>701</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>11878958_Online_Shoping_29.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>label_15</zorder>
     <zorder>tableView_2</zorder>
     <zorder>label_7</zorder>
     <zorder>ID_COMMANDE</zorder>
     <zorder>MONTANT</zorder>
     <zorder>label_8</zorder>
     <zorder>label_9</zorder>
     <zorder>label_10</zorder>
     <zorder>label_11</zorder>
     <zorder>pushButton_4</zorder>
     <zorder>pushButton_13</zorder>
     <zorder>pushButton_14</zorder>
     <zorder>DATE_COMMANDE</zorder>
     <zorder>pushButton_8</zorder>
     <zorder>comboBox</zorder>
     <zorder>STATUT</zorder>
     <zorder>label_13</zorder>
     <zorder>stats_commande</zorder>
     <zorder>statbtnC</zorder>
     <zorder>lineEdit_2</zorder>
     <zorder>triCB</zorder>
     <zorder>triC</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1151</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
